swagger_server/controllers/survey_response.py

```diff
@@ -35,7 +37,8 @@ def add_survey_response(self, case_id, file):
 
         if case_id and file and hasattr(file, 'filename'):
             file_name, file_extension = os.path.splitext(file.filename)
-            is_valid_file, msg = self._is_valid_file(file_name, file_extension)
+            file_contents = file.read()
```

Comment:
We probably want to use secure_filename here also. See http://werkzeug.pocoo.org/docs/0.12/utils/

Reply:
Done